src/easier68k/core/opcodes/move.py

```diff
@@ -87,60 +137,6 @@ def __str__(self):
         # Makes this a bit easier to read in doctest output
         return 'Move command: Size {}, src {}, dest {}'.format(self.size, self.src, self.dest)
 
-    @staticmethod
-    def is_valid(command: str, parameters: str) -> (bool, list):
-        """
-        Tests whether the given command is valid
-
-        >>> Move.is_valid('MOVE.B', 'D0, D1')[0]
-        True
-
-        >>> Move.is_valid('MOVE.W', 'D0')[0]
-        False
-
-        >>> Move.is_valid('MOVE.G', 'D0, D1')[0]
-        False
-
-        >>> Move.is_valid('MOVE.L', 'D0, A2')[0]
-        False
-
-        >>> Move.is_valid('MOV.L', 'D0, D1')[0]
-        False
-
-        >>> Move.is_valid('MOVE.', 'D0, D1')[0]
-        False
-
-        :param command: The command itself (e.g. 'MOVE.B', 'LEA', etc.)
-        :param parameters: The parameters after the command (such as the source and destination of a move)
-        :return: Whether the given command is valid and a list of issues/warnings encountered
-        """
-        issues = []
-        try:
-            parts = command.split('.')  # Split the command by period to get the size of the command
-            assert len(parts) <= 2, 'Unknown error (more than 1 period in command)'  # If we have more than 2 parts something is seriously wrong
-            assert parts[0].upper() == 'MOVE', 'Incorrect command passed in'
-            if len(parts) != 1:  # Has a size specifier
-                assert len(parts[1]) == 1, 'Size specifier must be 1 character'
-                assert parts[1] in Move.allowed_sizes, "Size {} isn't allowed for command {}".format(parts[1], command[0])
-
-            # Split the parameters into EA modes
-            params = parameters.split(',')
-            assert len(params) == 2, 'Must have two parameters'
-
-            src = EAMode.parse_ea(params[0].strip())  # Parse the source and make sure it parsed right
-            assert src.mode > EAMode.ERR, 'Error parsing src'  # -1 means error
-
-            dest = EAMode.parse_ea(params[1].strip())
-            assert dest.mode > EAMode.ERR, 'Error parsing dest'  # -1 means error
-
-            assert src.mode != EAMode.ARD, 'Invalid addressing mode'  # Only invalid src is address register direct
-            assert dest.mode != EAMode.ARD and dest.mode != EAMode.IMM, 'Invalid addressing mode'
-
-            return True, issues
-        except AssertionError as e:
-            issues.append((e.args[0], 'ERROR'))
-            return False, issues
-
     @staticmethod
     def get_word_length(command: str, parameters: str) -> (int, list):
```

Comment:
Is it possible to move this to a module-level method as well for consistency?

Reply:
yeah this looks like it should be fine as a module-level method